pkgs/build-support/setup-hooks/make-wrapper.sh

```diff
@@ -1,86 +1,99 @@
+#!bash
```

Comment:
Well, the shebang is pointless here. I guess running ShellCheck with "`-s bash`" would be better in this case. I certainly don't want a shebang that violates POSIX.